draft-meynell-panrg-scion-research-questions.md

```diff
@@ -196,9 +196,34 @@ This draft assumes the reader is familiar with some of the fundamental concepts
 
 ## ISD, AS identity
 
+The SCION protocol specifies 16 bits and 48 bits to identify the ISD and AS.
+This identification is, at this moment, necessary at the data-plane level.
+It is written in each one of the data packets to fully address the sender and receiver,
+as well as at the control-plane level, to identify the PCB sender and hops.
+
+While 48 bits seem sufficient to accommodate a large number of ASes,
+using 16 bits for the ISD might not be.
```

Comment:
using 16 bits for the ISD only offers 65,536 possible assignments which is conceivably insufficient.